webpack/client.config.js

```diff
@@ -20,14 +22,26 @@ module.exports = {
 					options: {
 						dev: isDev,
 						hydratable: true,
-						hotReload: true
+						hotReload: true,
+						emitCss: true
 					}
 				}
+			},
+			{
+				test: /\.css$/,
+				use: [
+					isDev ? 'style-loader' : MiniCssExtractPlugin.loader,
```

Comment:
This doesn't take into consideration source maps. That's probably something that should be out of the box.

Maybe something like:
```
use: [
          isDev
            ? { loader: 'style-loader', options: { sourceMap: true }}
            : MiniCssExtractPlugin.loader,
          { loader: 'css-loader', options: { sourceMap: isDev }}
        ],
```